apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -32,7 +32,7 @@ import { CardItemContentDirective } from '../../ui/card/card-item-content.direct
   imports: [CardComponent, ListItemComponent, CardItemContentDirective],
 })
 export class CityCardComponent implements OnInit {
-  cities: City[] = [];
+  cities = toSignal(this.store.cities$);
```

Comment:
That's great. You could have even used a signal inside the store directly instead of a behaviorSuject. With this works well 👍

Reply:
Got it - should be fixed now